main.go

```diff
@@ -45,8 +43,7 @@ func main() {
 
 	// Write log to file if the log flag was provided
 	if flags.Log != "" {
-		logFile, err := os.OpenFile(flags.Log, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666)
-		if err != nil {
+		if logFile, err := os.OpenFile(flags.Log, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666); err != nil {
```

Comment:
Did you test this code? I think you have a scope issue here.  I would expect Go to complain that logFile isn't defined on line 49.  See [this example](http://play.golang.org/p/v8RToYbPJn).